quickstarts/aws/aws-elemental-mediaconvert/config.yml

```diff
@@ -1,23 +1,16 @@
 id: f1079222-5bc9-41b0-9d6e-9cc2bf84b5b3
-slug: aws-elemental-mediapackage
+slug: aws-elemental-mediaconvert
```

Comment:
I would recommend not changing the slugs for quickstarts, as they can break any references in docs pages, developer pages, or I/O pages when the cache isn't properly cleared. 

Let me get clarification for you on this.

Reply:
Hi @josephgregoryii, if we see the entire config & dashboards, it's dependent on mediaconvert.
here is the AWS Document - https://docs.aws.amazon.com/mediaconvert/latest/ug/metrics.html